hmda/src/main/scala/hmda/api/http/filing/submissions/UploadHttpApi.scala

```diff
@@ -96,30 +98,31 @@ trait UploadHttpApi extends HmdaTimeDirectives {
                                submission,
                                uri)
                   } else {
-                    submissionNotAvailable(submissionId, uri)
+                    val errorResponse = ErrorResponse(
+                      404,
+                      s"Submissidson ${submissionId.toString} not available for upload",
+                      uri.path)
+                    complete(
+                      ToResponseMarshallable(
+                        StatusCodes.NotFound -> errorResponse)
+                    )
```

Comment:
I think this is worse than what we had before. The point of the `submissionNotAvailable` function is to extract this common boilerplate code to an object that can be brought in scope (`FilingResponseUtils`). It's possible that it needs refactoring or an additional method to account for this `404`, but I don't think introducing this back makes the code more readable or maintainable. To use this object in the check digit project as well, I would refactor `FilingResponseUtils` to the `common` project so it can be used from both places.